### DISCUSSION TITLE
Regarding attribute parsing from message body

### DISCUSSION THREAD

**Discussion by alfredbuehler** [creecros/Mailmagik#43] [General]
@creecros The problem: the user wants to create the task in column "Ready". But `&@column_id=2@&` fails, as 2 is the position rather than the id. The user cannot know the real id of a column. It's easy to transform the position to an id, but then the tag column_id no longer fits.
How should we handle this?

**Comment by creecros** [comment]
No, wait, I think I didn't understand. 

You're saying "position" is an attribute to the column table.

Maybe we just use `&@column_position=2@&`

And then map it the appropriate way during update.

We can make note of it in the readme.

**Reply by alfredbuehler** [reply]
Agree, but if we introduce own attribute names, why not just `column=<pos>|<name>`. This would IMHO be more preferably.

**Reply by creecros** [reply]
I cant argue with that, I like the idea
